index.html

```diff
@@ -11,13 +11,13 @@
 </head>
 <body>
   <div class="header">
-    <img id="store-logo" src="images/logo.png">
+    <a href="index.html"><img id="store-logo" src="images/logo.png"></a>
```

Comment:
This is going to give you an issue later on, You are going to be dynamically creating what the user sees based off which page is selected. Don't worry about it for now.